contracts/lazyclaim/IERC721LazyClaim.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/// @author: manifold.xyz
+
+/**
+ * Lazy Claim interface
+ */
+interface IERC721LazyClaim {
+    struct Claim {
+        bytes32 merkleRoot;
+        string uri;
+        uint totalMax;
+        uint walletMax;
+        uint startDate;
+        uint endDate;
+    }
+    function initializeClaim(address creatorContractAddress, bytes32 merkleRoot, string calldata uri, uint totalMax, uint walletMax, uint startDate, uint endDate) external;
```

Comment:
Change all uints to uint256 or uint (whatever we try to do to supercompress the struct)

Reply:
struct ClaimState {
 merkleRoot
 uri
  total;
 totalMax
 walletMax
 startDate
 endDate
 }

Reply:
Done